tests/chart_tests/test_astronomer_cp_dp_feature.py

```diff
@@ -13,6 +26,19 @@ class TestAstronomerCpDpFeature:
     def filter_charts_by_component(charts, component):
         return [chart for chart in charts if chart.get("metadata", {}).get("labels", {}).get("plane") == component]
 
+    @staticmethod
```

Comment:
I feel like we should move all these helper functions outside of the class. What do you think @Shubham-astro? IMHO it would make the code cleaner.